web/src/buddy_list.ts

```diff
@@ -80,10 +80,18 @@ type BuddyListRenderData = {
 function get_render_data(): BuddyListRenderData {
     const current_sub = narrow_state.stream_sub();
     const pm_ids_set = narrow_state.pm_ids_set();
+    const pm_ids_list = [...pm_ids_set];
+    let bot_count = 0;
+
+    for (const pm_id of pm_ids_list) {
+        if (people.is_valid_bot_user(pm_id)) {
+            bot_count += 1;
+        }
+    }
```

Comment:
What if we instead renamed `total_subscriber_count` to `total_human_subscribers_count` and adjusted the logic there to exclude bots like the current user?

I think that'd be a lot cleaner as an algorithm; after some variable renames this could be `other_users_count = total_human_users - total_human_subscribers`, which feels much more obviously correct.

Reply:
Posting in the #issues thread to discuss what we should do / your recommendation would have been the best approach to take here.

Reply:
> Playing with the UI some more, I think just not counting bots at all in the left sidebar makes sense. We just want the counts to line up with the lists.